CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+# Contributing Guidelines
+
+Thanks for interest in contributing to MTF.
+
+The following is a set of guidelines for contributing to MTF. These are not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## How to contribute to MTF
+
+### Reporting Bugs
+
+Before creating bug reports, please check a [list of known issues](https://github.com/fedora-modularity/meta-test-family/issues) to see if the problem has already been reported.
+
+If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/fedora-modularity/meta-test-family/issues/new). Be sure to include a **descriptive title and clear description** and please include **as many details as possible** to help maintainers reproduce the issue and resolve it faster. If possible, add a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
```

Comment:
Can you please add also info about version? We have several releases and it would be easy to find out what was changed since last tag.